jalil-portfolio.js

```
@@ -202,7 +202,304 @@ document.addEventListener('DOMContentLoaded', () => {
+    setTimeout(() => {
+        preloader.classList.add("hide");
+    }, 1200);
+});
+
+// =============================
+// MOBILE MENU
+// =============================
+const menuToggle = document.querySelector(".menu-toggle");
+const navLinks = document.querySelector(".nav-links");
+
+menuToggle.addEventListener("click", () => {
```

Biome (2.4.7)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)